src/geoparquet.py

```diff
@@ -108,7 +119,7 @@ def append(self, type, id, tags, wkb_hex):
         geom = shapely.wkb.loads(wkb_hex, hex=True)
         wkb = binascii.unhexlify(wkb_hex)
 
-        attrs = {key: tags.get(key) for key in self.tags}
+        attrs = {tag_name(key): tags.get(tag_name(key)) for key in self.tags}
```

Comment:
This seems like it'll need to be more flexible in the future, so that individual layer classes can specify how to produce each column value from a given feature's tags (since it won't be a 1:1 map anymore). Any thoughts on how to do this?

Reply:
One more note on this... The current solution makes a a tradeoff, offering essentially infinite flexibility (write code to produce whatever dictionary you want!), but it makes you write code to do it. An alternate approach would be to instead have a sort of high-level "mapping" description instead of code (i.e. a dict of regexes to.... idk what exactly...).

I think we don't actually know what we want in terms of a mapping description yet, which is why I went for the current approach. I have one use case (multilingual names), and can think of a few others with this general shape involving colons (for example, `addr:` tags), but it isn't entirely clear to me which direction everyone interested wants to go. We can always refine later as we discover more use cases.